docs/source/azure/index.rst

```diff
@@ -1,25 +0,0 @@
-.. _azure_overview:
-
```

Comment:
Did this file and the multisub files get moved also, or just removing them?

Reply:
So I'm removing this overview one entirely and it didn't have anything useful and Getting Started seems better.  The multi-sub is sort of renamed to "Advanced Usage" with the only topic currently covered in it being multi sub.